src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.Android.cs

```diff
@@ -107,6 +118,81 @@ private static Orientation DefaultDeviceOrientation
 				}, null);
 		}
 
+		partial void StartListeningOrientationChanged()
+		{
+			var mainLooper = Looper.MainLooper;
+			var context = ContextHelper.Current;
+
+			// Thread pool is used to avoid startup
+			// cost of threads creation.
+			ThreadPool.QueueUserWorkItem(
+				_ =>
+				{
+					if (Application.Context.GetSystemService(Context.SensorService) is SensorManager sensorManager)
+					{
+						_sensorManager = sensorManager;
+
+						var gravitySensor = _sensorManager.GetDefaultSensor(_gravitySensorType);
+
+						// Gravity sensor support seems to have been added in Nougat (Android 7), but not entirely functional.
+						// Therefore, only use it in Oreo+ 
+						var useGravitySensor = Build.VERSION.SdkInt >= BuildVersionCodes.O;
+
+						// If the the device has a gyroscope we will use the SensorType.Gravity, if not we will use single angle orientation calculations instead
+						if (gravitySensor != null && useGravitySensor)
+						{
+							_sensorEventListener = new OrientationListener(this);
+							_sensorManager.RegisterListener(_sensorEventListener, _sensorManager.GetDefaultSensor(_gravitySensorType), SensorDelay.Normal);
+						}
+						else
+						{
+							_orientationListener = new SimpleOrientationEventListener(orientation => OnOrientationChanged(orientation));
+							_contentObserver = new SettingsContentObserver(new Handler(mainLooper!), () => OnIsAccelerometerRotationEnabledChanged(IsAccelerometerRotationEnabled));
```

Comment:
Why is the main looper needed here?

Reply:
I just validated, and this logic was already there, this PR only moves it to a place where we can cancel the object initialization in case the sensor cannot start measuring allowing us to return null instead.

But looking at the code (and to answer your question): The  `ContentObserver` object init requires a Handler. It seems they wanted to use the MainThread looper to listen for these changes.